File: src/leetcode/2026/06/Solution1025.java
public class Solution1025 {
    public boolean divisorGame(int n) {
        return n % 2 == 0;
    }

    public static void main(String[] args) {
        Solution1025 solution = new Solution1025();
        System.out.println(solution.divisorGame(2)); // Output: true
        System.out.println(solution.divisorGame(3)); // Output: false
        System.out.println(solution.divisorGame(4)); // Output: true
    }
}

File: src/leetcode/2026/06/Solution1025.cpp
#include <iostream>
using namespace std;

class Solution1025 {
public:
    bool divisorGame(int n) {
        return n % 2 == 0;
    }
};

int main() {
    Solution1025 solution;
    cout << solution.divisorGame(2) << endl; // Output: true
    cout << solution.divisorGame(3) << endl; // Output: false
    cout << solution.divisorGame(4) << endl; // Output: true
    return 0;
}